src/main/java/vazkii/quark/oddities/container/EnchantmentMatrix.java

```diff
@@ -143,19 +143,17 @@ private EnchantmentDataWrapper generateRandomEnchantment(Map<Enchantment, Intege
 						}
 					}
 				}
-
+				
 				int valueAdded = getValue(enchantment, enchantLevel);
 				int currentValue = totalValue.getOrDefault(enchantment, 0);
-
-				if (valueAdded + currentValue > getValue(enchantment, enchantment.getMaxLevel()) +
-						getMaxXP(enchantment, enchantment.getMaxLevel()))
-					continue;
-
-				EnchantmentDataWrapper wrapper = new EnchantmentDataWrapper(enchantment, enchantLevel);
-				wrapper.normalizeRarity(influences, marked);
-				validEnchants.add(wrapper);
+				
+				if (valueAdded + currentValue <= getValue(enchantment, enchantment.getMaxLevel()) + getMaxXP(enchantment, enchantment.getMaxLevel())) {
+					EnchantmentDataWrapper wrapper = new EnchantmentDataWrapper(enchantment, enchantLevel);
+					wrapper.normalizeRarity(influences, marked);
+					validEnchants.add(wrapper);
+				}
```

Comment:
no unrelated changes please

Reply:
this was done because you can't continue in a forEach method

Reply:
nvm am low iq